### PR TITLE
Replaced Arrays with Maps in the json hystrix metrics snapshot as curl_json collectd plugin does not support arrays.

### PR DESCRIPTION
**Pull request by livingstonese**
Also excluding slf4j implementation from pom.

### CODE REVIEW

File: task/pom.xml [hunk 1]
```diff
@@ -109,6 +109,10 @@
                     <groupId>org.apache.httpcomponents</groupId>
                     <artifactId>httpcore</artifactId>
                 </exclusion>
+                <exclusion>
+                    <artifactId>slf4j-log4j12</artifactId>
```
Comment (kartikssj) [inline task/pom.xml]: Why is this needed?

Comment (livingstonese) [inline task/pom.xml]: The dependency brave-impl was added recently by Reghu and it has a dependency on slf4j-log4j12. Since this is an implementation of slf4j-api, we started getting error saying "Multiple implementations of slf4j found" and ServiceProxy was not running locally on Intellij Idea.
